libs/grunt-loader.js

```diff
@@ -0,0 +1,31 @@
+var fs = require('fs');
+var path = require('path');
+
+module.exports = {
+	/**
+	 * Given a folder, attempts to load any grunt modules included in the child directory node_modules. Due to flattening
+	 * in NPM 3+, grunt-springroll's new modules could actually be installed in the game's node_modules folder rather than
+	 * the node_modules folder underneath grunt-springroll. load-grunt-tasks attempts to connect the package.json with the
+	 * sibling folder node_modules, which is why tasks have failed to load NPM 3 and beyond. This however avoids that and
+	 * simply matches filenames based on convention
+	 *
+	 * @param {grunt} grunt The grunt instance with which to load tasks
+	 * @param {string} folder The folder to load tasks from (will look for a node_modules as a child directory)
+	 * @return void
+	 */
+	load: function(grunt, folder) {
+    var lastDirectory = process.cwd();
+
+    process.chdir(folder);
```

Comment:
It looks like this method has inconsistent indents, both in terms of depth and a mixture of tabs/spaces.